packages/core/workflows-sdk/src/utils/_playground.ts

```diff
@@ -45,13 +54,13 @@ const workflow2 = createWorkflow("workflow", function () {
   const workflowRes = workflow.asStep({ outsideWorkflowData: step1Res.step1 })
 
   return workflowRes
-})
+})*/
 
-workflow2()
-  .run({})
-  .then((res) => {
-    console.log(res.result)
-  })
+// workflow()
```

Comment:
question: Intentionally commented out?

Reply:
yes it is just a playground and I needed clarity :D